assets/queries/k8s/block_cluster_admin_rolebinding/query.rego

```diff
@@ -0,0 +1,16 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i]
+  metadata := resource.metadata
+  resource.kind == "ClusterRoleBinding"
+  resource.roleRef.name == "cluster-admin"
+
+	result := {
+    "documentId": input.document[i].id,
+    "searchKey": sprintf("metadata.name=%s.roleRef.name=cluster-admin", [metadata.name]),
+    "issueType": "IncorrectValue",
+    "keyExpectedValue": "Ensure that the cluster-admin role is only used where required (RBAC)",
```

Comment:
Change keyExpectedValue to:
`sprintf("Resource name '%s' of kind '%s' isn't binding 'cluster-admin' role with superuser permissions", [metadata.name, resource.kind])`

Reply:
ok!